### PR TITLE
accordion

### PR DESCRIPTION
**Pull request by dkang23**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,14 +1,14 @@
 import React from 'react';
 import Button from '@material-ui/core/Button';
 import IssuesPage from './components/IssuesPage.jsx';
-import UserData from './components/UserData/UserData.jsx'
+import UserData from './components/UserData/UserData.jsx';
```
Comment (ghost) [inline src/App.js]: File extensions should not be included in the import unless multiple files in a directory have the same file name.

File: src/components/IssuesPage.jsx [hunk 2]
```diff
@@ -1,12 +1,41 @@
...
+  ];
+
   return (
     <div>
       {/* <Statuses /> */}
-      <IssuesList
-      // issues={props.}
-      />
+      <IssuesList issues={example} />
     </div>
```
Comment (ghost) [inline src/components/IssuesPage.jsx:39]: Consider moving example outside the scope of IssuesPage and then making your `() => {};` function a `() => ();` function due to instant return.

Comment (dkang23) [reply src/components/IssuesPage.jsx:39]: fixed requested changes

Comment (Joed11) [reply src/components/IssuesPage.jsx:39]: Thanks for adjusting!

**Pull request merged**